applications/webhooks/src/webhooks_shared_listener.erl

```diff
@@ -297,7 +297,12 @@ handle_info(_Info, State) ->
 %% @end
 %%--------------------------------------------------------------------
 handle_event(_JObj, _State) ->
-    {'reply', []}.
+    WHZones = lists:sort([kz_util:to_binary(Z)
+                          || Z <- kz_nodes:whapp_zones(?APP_NAME)
+                         ]),
+    Options = [{<<"webhooks_zones">>, WHZones}
+              ] ++ should_handle_federated_events(WHZones),
```

Comment:
@icehess since we're doing all logic to have a 'should_handle_federated', there's no need to include 'zones' in Props.
this could be as simple as [{'should_handle_federated', should_handle_federated_events(WHZones)}] or
{'reply', should_handle_federated_events(WHZones)}.